paper/main.tex

```diff
@@ -193,15 +201,16 @@ \section{Background}
 \begin{itemize}
     \item \st{Do not be envious} Be a little bit envious
     \item Be ``nice'' in non-noisy environments or when game lengths are longer
-    \item Reciprocate both cooperation and defection correctly; Be somewhat provocable
+    \item Reciprocate both cooperation and defection correctly; Be provocable and generous
```

Comment:
Maybe we can soften to something like "Reciprocate both cooperation and defection" appropriately: be provocable and generous but not extremely